package.json

```diff
@@ -4,7 +4,8 @@
   "main": "index.js",
   "scripts": {
     "start": "http-server .",
-    "build": "webpack"
+    "build": "webpack",
+    "lint": "eslint ."
```

Comment:
```suggestion
    "lint": "eslint --ext .cjs,.js ."
```

Reply:
It's weird that's still not the default for eslint...but maybe that's changed in v9. Will fix.